pkg/identity/identity.go

```diff
@@ -50,5 +53,30 @@ func ConfigValidate(identitySpec kotsv1beta1.IdentityConfigSpec, ingressSpec kot
 		return errors.New("identityServiceAddress required or ingressConfig.ingress must be enabled")
 	}
 
+	if err := evaluateDexConnectorsValue(ctx, namespace, &identitySpec.DexConnectors); err != nil {
+		return errors.Wrap(err, "failed to evaluate dex connectors value")
+	}
+
+	// validate issuers
```

Comment:
id be in favor of only doing this in the route and leaving it as is from the cli.

1. the user will not get locked out of the cli if its misconfigured
2. the cli is an escape hatch if there is an issue with this approach